src/pudl/metadata/sources.py

```diff
@@ -835,5 +835,44 @@
         "license_raw": LICENSES["us-govt"],
         "license_pudl": LICENSES["cc-by-4.0"],
     },
+    "vceregen": {
+        "title": "Vibrant Clean Energy Renewable Generation Profiles",
+        "path": "https://vibrantcleanenergy.com/wp-content/uploads/2020/08/VCE-Weather-Dataset-Overview_August2020.pdf",
+        "description": (
```

Comment:
Could make this a path to the internal file, but wasn't 100% sure how to do that.